<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="menuFrame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTreeWidget" name="menuTree">
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
         <property name="columnCount">
          <number>1</number>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>Accueil</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fiches chiens</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Registres</string>
          </property>
          <item>
           <property name="text">
            <string>Entrées/Sorties</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Garderie</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Adhérents</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>Vétérinaire</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Autres</string>
          </property>
          <item>
           <property name="text">
            <string>Liste rouge adoptants</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Animaux perdus</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Demandes d'adoption</string>
           </property>
          </item>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="currentIndex">
       <number>8</number>
      </property>
      <widget class="QWidget" name="homePage"/>
      <widget class="QWidget" name="dogSheetsPage"/>
      <widget class="QWidget" name="entryRegistryPage"/>
      <widget class="QWidget" name="careRegistryPage"/>
      <widget class="QWidget" name="redListPage"/>
      <widget class="QWidget" name="lostPage"/>
      <widget class="QWidget" name="adoptionDemandPage"/>
      <widget class="QWidget" name="membersPage"/>
      <widget class="QWidget" name="vetPage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>912</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
